src/widgets/dialogs/UserInfoPopup.cpp

```diff
@@ -384,6 +388,21 @@ void UserInfoPopup::updateUserData()
 {
     std::weak_ptr<bool> hack = this->hack_;
 
+    const auto onIdFetchFailed = [this]() {
+        // this can occur when the account doesn't exist.
+        this->ui_.followerCountLabel->setText(TEXT_FOLLOWERS +
+                                              QString(TEXT_UNAVAILABLE));
+        this->ui_.viewCountLabel->setText(TEXT_VIEWS +
+                                          QString(TEXT_UNAVAILABLE));
+        this->ui_.createdDateLabel->setText(TEXT_CREATED +
+                                            QString(TEXT_UNAVAILABLE));
+
+        this->ui_.nameLabel->setText(this->userName_ +
+                                     QString("(account doesn\'t exist)"));
+        this->ui_.userIDLabel->setText(QString("ID is not available."));
+        this->ui_.userIDLabel->setProperty("copy-text",
+                                           QString("ID is not available."));
```

Comment:
Why are we using a different error here than in the 3 other labels?

Reply:
Using only one now.